ansible_events/app.py

```diff
@@ -57,7 +69,10 @@ async def run(parsed_args) -> None:
     logger.info("Cancelling event source tasks")
     for task in tasks:
         task.cancel()
-    await asyncio.gather(*tasks)
+    try:
+        await asyncio.gather(*tasks)
```

Comment:
I have doubts about this particular logic. If first task raises the `CancelledError`, the remaining tasks may not be cleaned up properly. I need to verify this.

Reply:
UPD: The `CancelledError` is raised out of `asyncio.gather()` after all all tasks completed, so the described above behaviour is not an issue. However it may be a good idea to call `asyncio.gather(*tasks, return_exceptions=True)` to log any exception that is distinct from `CancelledError`.